spec/services/subscriptions/create_service_spec.rb

```diff
@@ -482,6 +482,31 @@
             end
           end
 
+          context 'when old subscription was payed in advabce' do
```

Comment:
`advance` You were testing us here to see if we are going through it carefully :D